### DISCUSSION TITLE
Implementation of heroes and garden warfare classes?

### DISCUSSION THREAD

**Discussion by Lhruggles** [GrassyDev/PvZCubed#16] [Ideas]
I was wondering if GW classes and the heroes would be playable in some way or if they will be mobs? It would be cool to play as them honestly.

**Comment by GrassyDev** [comment]
I still don't know what I'm gonna do with them. Any suggestions would be welcome!

**Reply by Lhruggles** [reply]
I feel at the very least an I Zombie mode would be cool, allowing you to become one of the hero/Garden warfare classes and join the horde. Well, maybe not all of them, possibly just Super Brainz, Electric slide and imp, as they’re the most used/popular. Idk how to implement player plants as well, as many are not humanoid.

**Reply by Lhruggles** [reply]
For playing as the zombies I think it could work kinda like the vampirism mod, letting players equip uniqie weapons and giving a few skills. Imp would obviously have a mech as it’s skill and guns as weapons, and electric slide could probably call backup dancers, as for super brainz maybe he could have limited flight and better default stats.
